### DISCUSSION TITLE
Do you use GitHub Discussions?

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9534] [Rules Discussions]
Secret Ingredients to Quality Software | SSW Rules provides best practices for developing secure, reliable, and efficient .NET, Azure, CRM, Angular, React, Dynamics, and AI applications. Learn more today!

https://www.ssw.com.au/rules/use-github-discussions/

**Comment by tiagov8** [comment]
I've experienced a scenario where I felt my new Issue wasn't in the right place because it was more of a question than a request. Now I know where it belonged to :)

**Comment by wicksipedia** [comment]
I'm a fan of using GitHub discussions to discuss/make decisions as opposed to email - primarily because email is a data island. IMHO this is a cleaner way of organizing the conversation
